helpers/upgrading.py

```diff
@@ -146,7 +148,24 @@ def new_terminology(upgraded_dict):
         return upgraded_dict
 
     @staticmethod
-    def two_databases(upgraded_dict, current_dict):
+    def set_compose_version(upgraded_dict: dict) -> dict:
+        try:
+            upgraded_dict['compose_version']
+        except KeyError:
+            # FIXME On macOS, Docker Desktop always installs a symlink for
+            #   `docker-compose`. Version will be always detected as v1 even if
+            #   user has chosen v2 in Docker Desktop preferences.
+            if which('docker-compose') is None:
+                compose_version = 'v2'
+            else:
+                compose_version = 'v1'
```

Comment:
Is there is a reason for using `try...except` here? Maybe something more compact:
```python3
if 'compose_version' not in upgraded_dict:
    compose_version = 'v2' if which('docker-compose') is None else 'v1'
```

Reply:
There is a reason, but not a good one :-P